packages/core/src/overlay/OverlayPanelCloseButton.css

```diff
@@ -11,5 +11,10 @@
 }
 
 .saltOverlayPanelCloseButton ~ .saltOverlayPanelContent {
-  padding-right: var(--salt-spacing-400);
+  /* (button-padding * 2) + button-icon-size + (button-borderWidth * 2) */
+  --saltOverlayCloseButton-width: calc(
```

Comment:
This feels like a pretty obtuse `calc`. Let me know if there's a better way to get the exact button width.

Reply:
Haven't looked existing code. if close button will be on its own space, shall we consider using `display:grid` with the button on its own column

Reply:
I did try a version that used the button's intrinsic size (I used flex rather than grid, but same concept). The problem was that it pushed the content area away from the right border (i.e. inserting a column for the button) so the scrollbar for long content ended up in the middle of the overlay rather than up against the side.

Keeping the scroll bar anchored on the side of the overlay was deemed better (discussed this with Ben), so the complicated calc is actually the simpler of the options.

Reply:
When i look at [example](https://cf73ea00.saltdesignsystem-storybook.pages.dev/?path=/story/core-overlay--close-button) with a lot of content and without title, it's shown "content | scrollbar | close"?